notifications_python_client/notifications.py

```diff
@@ -26,3 +26,18 @@ def send_email_notification(self, to, template_id, personalisation=None):
 
     def get_notification_by_id(self, id):
         return self.get('/notifications/{}'.format(id))
+
+    def get_all_notifications(self, status=None, template_type=None):
+        data = {}
+        if status:
+            data.update({
+                'status': status
+            })
+        if template_type:
+            data.update({
+                'template_type': template_type
+            })
```

Comment:
`data = {key: eval(key) for key in ['status', 'template_type'] if eval(key)}` :wink: